lib/Model/ParameterSet.php

```diff
@@ -12,33 +12,25 @@
 
 namespace PhpBench\Model;
 
-use ArrayObject;
-
-/**
- * @extends ArrayObject<string,mixed>
- */
-class ParameterSet extends ArrayObject
+final class ParameterSet
```

Comment:
revert final, make it `/** final */`

Reply:
need to "break" B/C on this class anyway as the constructor requires different arguments.